examples/inverse/plot_resolution_metrics.py

```diff
@@ -0,0 +1,127 @@
+# forward operator with fixed source orientations
+forward = mne.convert_forward_solution(forward, surf_ori=True,
+                                       force_fixed=True)
+
+# noise covariance matrix
+noise_cov = mne.read_cov(fname_cov)
+
+# evoked data for info
+evoked = mne.read_evokeds(fname_evo, 0)
+
+# make inverse operator from forward solution
+# free source orientation
+inverse_operator = mne.minimum_norm.make_inverse_operator(
+    info=evoked.info, forward=forward, noise_cov=noise_cov, loose=0.,
+    depth=None)
+
+# regularisation parameter
+snr = 3.0
+lambda2 = 1.0 / snr ** 2
+
+# compute resolution matrix for MNE
+rm_mne = make_resolution_matrix(forward, inverse_operator,
+                                method='MNE', lambda2=lambda2)
+
+# compute resolution matrix for sLORETA
+rm_dspm = make_resolution_matrix(forward, inverse_operator,
+                                 method='dSPM', lambda2=lambda2)
+
+# Compute peak localisation error (PLE) for PSFs
+ple_mne_psf = resolution_metrics(rm_mne, inverse_operator['src'],
+                                 function='psf',
+                                 kind='localization_error', metric='peak')
+ple_dspm_psf = resolution_metrics(rm_dspm, inverse_operator['src'],
+                                  function='psf',
+                                  kind='localization_error', metric='peak')
+
+# Compute spatial deviation (SD) for PSFs
+sd_mne_psf = resolution_metrics(rm_mne, inverse_operator['src'],
+                                function='psf',
+                                kind='spatial_extent', metric='sd')
+sd_dspm_psf = resolution_metrics(rm_dspm, inverse_operator['src'],
+                                 function='psf',
+                                 kind='spatial_extent', metric='sd')
+
+# Visualise peak localisation error (PLE) across the whole cortex for PSF
+
+brain_ple_mne = ple_mne_psf.plot('sample', 'inflated', 'lh',
+                                 subjects_dir=subjects_dir, figure=1,
+                                 clim=dict(kind='value', lims=(0, 2, 4)))
+brain_ple_mne.add_text(0.1, 0.9, 'PLE MNE', 'title', font_size=16)
+
+brain_ple_dspm = ple_dspm_psf.plot('sample', 'inflated', 'lh',
+                                   subjects_dir=subjects_dir, figure=2,
+                                   clim=dict(kind='value', lims=(0, 2, 4)))
+brain_ple_dspm.add_text(0.1, 0.9, 'PLE dSPM', 'title', font_size=16)
+
+# Subtract the two distributions and plot this difference
+diff_ple = ple_mne_psf - ple_dspm_psf
+
+brain_ple_diff = diff_ple.plot('sample', 'inflated', 'lh',
+                               subjects_dir=subjects_dir, figure=3,
+                               clim=dict(kind='value', pos_lims=(0., 1., 2.)))
+brain_ple_diff.add_text(0.1, 0.9, 'PLE MNE-dSPM', 'title', font_size=16)
+
+print('dSPM has generally lower peak localization error (red color) than MNE \
+       in deeper brain areas, but higher error (blue color) in more \
+       superficial areas.')
```

Comment:
This print statement is odd in two ways. Using the line continuation operator `\` means that the indentation spaces on the following line become part of the string, so you end up with one space between most pairs of words, but eight spaces between some pairs of words.  Python allows strings to be spread over multiple lines like this:

```
print('foo'
      'bar '
      'and baz.')
```
yielding `foobar and baz.` when printed. Note the lack of commas and the explicit inclusion of a space after `bar` and not after `foo` (EDIT: you seem to know this already, since you do it in later `print` statements).  That said, the second way these are odd is that these useful, informative statements are getting lost among the auto-generated output from the code.  These should be expressed as rendered text in between code blocks. To do that, add this after line 100:

```

###############################################################################
# The plots show that dSPM has generally lower peak localization error (red
# color) than MNE in deeper brain areas, but higher error (blue color) in more
# superficial areas.
```

Then do the same thing for the other print statement at the end of the tutorial.